app/views/users/profile.html.erb

```diff
@@ -19,7 +19,7 @@
 		<%= f.input :birthdate, :label => "Birth Date", :start_year => Time.now.year-13, :end_year => 1930 %>
         <%= f.input :other_languages, :label => "Other languages you speak", :hint => "If you speak any languages other than English, enter them here to help coordinate event planning and project assignment." %>
 		
-		<%- if @current_user.can_edit?(@person, :high_school_id) && @person.is_a?(Participant) -%>
+		<%- if @current_user.can_edit?(@person, :high_school_id) && @person.respond_to?(:high_school) -%>
```

Comment:
@aayush19saxena I made a brief change here to make this more flexible, since the user could be a `Student` or a `Participant`. So instead of explicitly checking if the person is a participant, we can check if the person is capable of responding to `high_school`.